src/raftstore/store/store.rs

```diff
@@ -1863,6 +1864,7 @@ fn verify_and_store_hash(region_id: u64,
               expected_index);
     }
 
+    info!("[region {}] save hash of {} for consistency check later.", region_id, expected_index);
```

Comment:
No, if they are different, L1845 will show them all. If they are the same, we still can infer it from the Verify Admin Command log.